### PR TITLE
modified one line in .get_timepoints_survsummary, to round cum.n.event

### PR DESCRIPTION
**Pull request by LaurenSamuels**
Hi- thank you so much for this package. I added rounding in one line; you might also want to modify the line for cum.n.censor the same way.

### CODE REVIEW

**Comment by MaverickMeerkat** [comment]
+1; Why is this not merged? This really prevents use of ggsurvplot when wanting to present cumulative number of events :-(

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 .DS_Store
 docs/.DS_Store
 inst/doc
+*.swp
```
Comment (MaverickMeerkat) [inline .gitignore:7]: This seems to be not needed....

**Review by MaverickMeerkat — CHANGES_REQUESTED** [review]
The git ignore change is probably a mistake. Could you please undo it? 
Also you added a binary file which not sure if needed.
Maybe this is what preventing this important pull request from being approved...

**Comment by LaurenSamuels** [comment]
@MaverickMeerkat , Thank you for taking a look at this. I agree that the gitignore change should not be included, but I am having trouble undoing it. Would you be able to either spell out exactly what I need to do, or just add the one useful line to .get_timepoints_survsummary another way?

**Comment by MaverickMeerkat** [comment]
I opened a new pull request, because otherwise I think you would have needed to grant me permissions to change your own PR. It's here: https://github.com/kassambara/survminer/pull/564

**Comment by LaurenSamuels** [comment]
Closing this request, because @MaverickMeerkat has submitted a better one (many thanks!).

**Pull request closed without merging**